### PR TITLE
Proofreading Melinda Seckington captions 00:45 - 01:00

### PR DESCRIPTION
**Pull request by davidszotten**

### CODE REVIEW

File: 6/melinda-seckington-captions.srt [hunk 1]
```diff
@@ -2319,7 +2319,7 @@ DN: What is it calling merge anyway?
 
 552
 00:49:38,712 --> 00:49:40,960
-We're up to 73.
+Parameters is 73.
```
Comment (nelstrom) [inline 6/melinda-seckington-captions.srt:2322]: :+1:

File: 6/melinda-seckington-captions.srt [hunk 2]
```diff
@@ -2327,7 +2327,7 @@ MS: Well that's because...
 
 554
 00:49:48,304 --> 00:49:50,736
-that's the only pre answer it has.
+that's the only params that it has.
```
Comment (nelstrom) [inline 6/melinda-seckington-captions.srt:2330]: :+1:

**Comment by nelstrom** [comment]
Great job @davidszotten, thanks!

**Comment by davidszotten** [comment]
np

**Pull request merged**